internal/notifier/bitbucketserver.go

```diff
@@ -258,20 +260,37 @@ func (b BitbucketServer) postBuildStatus(ctx context.Context, rev, state, name,
 }
 
 func (b BitbucketServer) createApiPath(rev string) string {
-	return fmt.Sprintf(bbServerEndPointTmpl, b.ProjectKey, b.RepositorySlug, rev)
+	return fmt.Sprintf(bbServerEndPointTmpl, b.ContextPath, b.ProjectKey, b.RepositorySlug, rev)
 }
 
-func parseBitbucketServerGitAddress(s string) (string, string, error) {
-	host, id, err := parseGitAddress(s)
+func parseBitbucketServerGitAddress(s string) (string, string, string, error) {
+	scheme := strings.Split(s, ":")[0]
+	if scheme != "http" && scheme != "https" {
+		return "", "", "", fmt.Errorf("could not parse git address: unsupported scheme type in address: %s. Must be http or https", scheme)
+	}
+
+	host, idWithContext, err := parseGitAddress(s)
 	if err != nil {
-		return "", "", fmt.Errorf("could not parse git address: %w", err)
+		return "", "", "", fmt.Errorf("could not parse git address: %w", err)
+	}
+
+	idWithContext = "/" + idWithContext
+
+	// /scm/ is always part of http/https clone urls : https://community.atlassian.com/t5/Bitbucket-questions/remote-url-in-Bitbucket-server-what-does-scm-represent-is-it/qaq-p/2060987
+	lastIndex := strings.LastIndex(idWithContext, "/scm/")
+	if lastIndex < 0 {
+		return "", "", "", fmt.Errorf("could not parse git address: supplied provider address is not http(s) git clone url")
```

Comment:
I agree, I'll address this.

Reply:
Done

Reply:
If the user goes to the UI and piclup the https git clone url, then it is guaranteed to have SCM in it. The error states that the user did not supply the git clone url. Typically users will pick up the clone url from the UI. 

Stating /scm/ in there may confuse some because they may not be aware of what /scm/ is. I am not sure how to be explicit here without being too explicit about /scm/ :)

Let me know how you'd like it and I'll implement it accordingly.